test/functional/Java19andUp/src/org/openj9/test/jep425/VirtualThreadTests.java

```diff
@@ -150,7 +150,17 @@ private static int readVirtualThreadStates(Class vthreadCls, String fieldName) t
 	@Test
 	public void test_verifyJVMTIMacros() {
 		final int JVMTI_VTHREAD_STATE_NEW = 0;
+		final int JVMTI_VTHREAD_STATE_STARTED = 1;
+		final int JVMTI_VTHREAD_STATE_RUNNABLE= 2;
+		final int JVMTI_VTHREAD_STATE_RUNNING = 3;
+		final int JVMTI_VTHREAD_STATE_PARKING = 4;
+		final int JVMTI_VTHREAD_STATE_PARKED = 5;
+		final int JVMTI_VTHREAD_STATE_PINNED = 6;
+		final int JVMTI_VTHREAD_STATE_YIELDING = 7;
 		final int JVMTI_VTHREAD_STATE_TERMINATED = 99;
+		final int JVMTI_VTHREAD_STATE_SUSPENDED = (1<<8);
```

Comment:
formatting nitpick: missing space.
```suggestion
		final int JVMTI_VTHREAD_STATE_SUSPENDED = (1 << 8);
```

Reply:
Updated.